apps/main/src/lend/lib/apiLending.ts

```diff
@@ -2002,7 +2006,7 @@ function _sortBands(bandsBalances: { [index: number]: { borrowed: string; collat
   return { bandsBalancesArr, bandsBalances }
 }
 
-async function _fetchChartBandBalancesData(
+export async function _fetchChartBandBalancesData(
```

Comment:
Some of these functions are only used in `user-loan-details.ts`. I don't mind if we moved them over there as to shrink the usage of this giant global class.